src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphEntityDAO.scala

```diff
@@ -11,6 +11,7 @@ import scala.collection.JavaConversions._
 import scala.collection.JavaConverters._
 
 import org.broadinstitute.dsde.rawls.model.AttributeConversions._
+import CachedTypes._
```

Comment:
if it is forgotten it will not compile and complain that it can't find an implicit